src/style.css

```diff
@@ -0,0 +1,83 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("/fonts/Avenir.ttc"),
+    url("fonts/Avenir-Book.ttf"),
+    url("fonts/Avenir-Heavy.ttf");
+}
+
+body {
+  box-sizing: border-box;
+  font-weight: 300;
+  margin: 0;
+}
+
+.serchBig {
+  width: 900px;
+  height: 70px;
```

Comment:
Don't use fixed width (block should have 100% width)

Reply:
done